src/gui/MainWindow.cpp

```diff
@@ -841,6 +839,8 @@ void MainWindow::applySettingsChanges()
         m_inactivityTimer->deactivate();
     }
 
+    m_ui->toolBar->setHidden(config()->get("GUI/HideToolbar").toBool());
```

Comment:
add a second parameter to the get function of "false" or add it to the default config entries in config.cpp

Reply:
Done.